protocol/chainlib/jsonRPC.go

```diff
@@ -61,7 +61,12 @@ func (apip *JsonRPCChainParser) getSupportedApi(name, connectionType string, int
 
 func (apip *JsonRPCChainParser) CraftMessage(parsing *spectypes.ParseDirective, connectionType string, craftData *CraftData, metadata []pairingtypes.Metadata) (ChainMessageForSend, error) {
 	if craftData != nil {
-		chainMessage, err := apip.ParseMsg("", craftData.Data, craftData.ConnectionType, metadata, extensionslib.ExtensionInfo{LatestBlock: 0})
+		path := craftData.Path
```

Comment:
not sure why path is here can you explain why we moved from "" to path / internal path ? 
(internal path part makes sense but path doesnt )

Reply:
That's because when we send node messages on init to each node in the node_urls, we want to use the right internal path. 
You can look the use here:

https://github.com/lavanet/lava/blob/eba62e6063f38f5f9cec6d259435d295a25342a3/protocol/chainlib/chain_fetcher.go#L170